src/Data/Show.purs

```diff
@@ -42,32 +48,45 @@ instance showArray :: Show a => Show (Array a) where
 instance showProxy :: Show (Proxy a) where
   show _ = "Proxy"
 
+instance showVoid :: Show Void where
+  show = absurd
+
 instance showRecord ::
   ( Nub rs rs
   , RL.RowToList rs ls
   , ShowRecordFields ls rs
   ) =>
   Show (Record rs) where
   show record = case showRecordFields (Proxy :: Proxy ls) record of
-    [] -> "{}"
-    fields -> intercalate " " [ "{", intercalate ", " fields, "}" ]
+    "" -> "{}"
+    r -> "{ " <> r <> " }"
```

Comment:
```purs
show record = "{" <> showRecordFields (Proxy :: Proxy ls) record) <> "}"
```

Reply:
You are right, that does work and is a simpler way to accomplish the same thing. I have made the changes